log-base/src/Log/Monad.hs

```diff
@@ -42,6 +45,24 @@ newtype LogT m a = LogT { unLogT :: InnerLogT m a }
   deriving (Alternative, Applicative, Functor, Monad, MonadBase b, MonadCatch
            ,MonadIO, MonadMask, MonadPlus, MonadThrow, MonadTrans)
 
+instance MonadError e m => MonadError e (LogT m) where
```

Comment:
Given that we have GND enabled, can't we avoid writing these instances by hand by mentioning them in the `derving` clause above?